en/advanced_config/advanced_mc_position_tuning.md

```diff
@@ -1,96 +1,111 @@
-<!-- Fixes page load bug in notation, courtesy https://github.com/GitbookIO/plugin-mathjax/issues/34#issuecomment-349453673 -->
-<script>
-gitbook.events.bind("page.change", function() {
-    MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
-}
-</script>
-
 # Advanced Multicopter Position Control Tuning
- The objective of this document is to give an overview of the higher-level position-control tuning parameters for multicopters. Higher-level tuning applies to any parameter that has an effect on
- the desired position/velocity setpoint as opposed to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
- The set of higher-level tuning parameters can be split into two sections: tunig parameters for manual control position mode and tuning parameters for auto. Some parameters will have an effect on both
- modes. 
 
-> **Warning** This guide is for advanced users/experts. 
+This document provides an overview of the multicopter position-control tuning parameters that affect the value of a desired setpoint (as contrasted with those parameters that affect how well the vehicle tracks the setpoint). 
+
+> **Warning** This guide is for advanced users/experts.
+
+<span></span>
+> **Tip** Follow the instructions in the [Multicopter PID Tuning Guide](../advanced_config/pid_tuning_guide_multicopter.md) *before* doing any of the higher-level related control tuning described here. Do not use the advanced position control tuning parameters to fix bad tracking or vibration!
+
+## Overview
+
+The input to the P/PID controller is a *desired setpoint* that the vehicle should attempt to track. [PID Tuning](../advanced_config/pid_tuning_guide_multicopter.md) ("Lower level" tuning) aims reduce the error between the desired setpoint and the estimate of the vehicle state. Poor P/PID Gains can lead to instability.
```

Comment:
@Stifael Thanks for explanation. I have updated the overview to provide a definition of the different types of tuning. To make this work I did an "implicit" renaming:
- Lower:  PID tuning, 
- Higher: Setpoint value tuning
I think that is more intuitive. 

Anyway, can you please read this introduction section and confirm it is OK. In particular, are there any other impacts to poor tuning of the setpoint value?

Reply:
yes, correct. But is it not "aims at ..."?

The only impact for poor setpoint tuning are the extremes: very jerky or very slow/smooth (with large delay in direction) flight motion

Reply:
"aims to". I'll fix this.